mission/config/subconfigs/vehicle_respawn_info.hpp

```diff
@@ -933,4 +978,63 @@ class spawn_point_types {
 			};
 		};
 	};
+
+	class boat_light {
+		name = "Light Boats";
```

Comment:
These are all STAB boats, so might make a bit more sense to have it like this
```suggestion
	class boat_stab {
		name = "Boats - STAB";
```